### PR TITLE
For #19113 - Change topSite corner radius to 8dp

### LINKED ISSUE #19113: Change border radius for top sites icon from 4dp to 8dp
### Acceptance Criteria
- Based on Figma, the border radius for the top sites icon should be 8dp https://www.figma.com/file/lfgj15zbNkL9Pp4hdZGJgp/Proton-Mobile-UI?node-id=809%3A1
- We decided to create a separate issue for this since it is not part of the acceptance criteria for adding shadows/elevation

### PR DESCRIPTION
**Pull request by codrut-topliceanu**
Fixes #19113

### Pull Request checklist

- [x] **Tests**: This PR includes no tests as it is only a minor change.
- [x] **Screenshots**: This PR includes screenshots.
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md).

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

Before:
image

After:
image

### CODE REVIEW

**Review by gabrielluong — APPROVED** [review]
Thanks! This looks good to me. Let's wait for @brampitoyo's UX review before landing.

**Review by brampitoyo — APPROVED** [review]
r+!

**Pull request merged**
